### PR TITLE
Mod application: Quickjoin

### PR DESCRIPTION
**Pull request by QWERTZexe**
Quickjoin is a very useful mod which allows you to join any gamemode from any game on hypixel in a matter of seconds!

### CODE REVIEW

File: files/discords.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
         "fancyname": "Prism Launcher",
         "nicknames": ["prism community"]
     },
+    {
+        "id": "quickjoin",
+        "code": "Vp6Q4FHCzf",
+        "icon": "https://cdn.glitch.global/d204ee01-249d-4fd8-8b05-bea32f8a9bfa/standard%20(3).png",
```
Comment (MicrocontrollersDev) [inline files/discords.json]: icons should be stored in the repo itself, see https://github.com/SkyblockClient/SkyblockClient-REPO/tree/main/files/icons

**Comment by QWERTZexe** [comment]
Ok @MicrocontrollersDev I fixed the icon 👍

**Comment by MicrocontrollersDev** [comment]
I have no clue if the animated png is gonna work and i dont remember how to set up XAMPP to test, will leave review for someone else. Let's ping KTi

File: files/discords.json [hunk 2]
```diff
@@ -15,6 +15,14 @@
...
         "nicknames": ["prism community"]
     },
+    {
+        "id": "quickjoin",
+        "code": "Vp6Q4FHCzf",
+        "icon": "quickjoin.png",
+        "description": "The inofficial support server for QWERTZmods",
+        "nicknames": ["qj"],
+        "mods": ["quickjoin"]
+    },
```
Comment (KTibow) [inline files/discords.json]: Where friendly name

File: files/mods.json [hunk 3]
```diff
@@ -164,6 +164,29 @@
         "packages": ["potioneffects", "redaction", "polyfrostkeystrokes"],
         "hash": "8e3980e5abaf1c5c5adb4dae6f3ed9f4"
     },
+    {
+        "id": "quickjoin",
+        "display": "Quickjoin",
+        "creator": "QWERTZ_EXE",
```
Comment (KTibow) [inline files/mods.json:170]: Please be consistent with other platforms in how you style your name

File: files/mods.json [hunk 4]
```diff
@@ -164,6 +164,29 @@
...
+        "description": "Quickly join any game on Hypixel!",
```
Comment (KTibow) [inline files/mods.json]: Please don't use exaggerating punctuation, and make this more descriptive - how does this mod accomplish the goal? It adds a menu, right? So say that.

File: files/mods.json [hunk 5]
```diff
@@ -164,6 +164,29 @@
...
+                "text": "GitHub",
+                "link": "https://github.com/QWERTZexe/Quickjoin"
+            },
+            {
+                "icon": "modrinth.png",
+                "text": "Modrinth",
+                "link": "https://modrinth.com/mod/quickjoin"
+            }
+        ],
+        "categories": ["Other"],
```
Comment (KTibow) [inline files/mods.json]: Pretty sure this isn't a valid category

**Review by KTibow — DISMISSED** [review]
And re: icon (GH mobile isn't letting me comment on the file) 

This icon is oversensational and out of place compared to other icons, consider replacing it with a more functional one (identifiable at small sizes, static, possibly uses graphic elements)

File: files/discords.json [hunk 6]
```diff
@@ -15,6 +15,14 @@
         "fancyname": "Prism Launcher",
         "nicknames": ["prism community"]
     },
+    {
+        "id": "quickjoin",
+        "code": "Vp6Q4FHCzf",
+        "icon": "quickjoin.png",
+        "description": "The inofficial support server for QWERTZmods",
```
Comment (MicrocontrollersDev) [inline files/discords.json]:
```suggestion
        "description": "The unofficial support server for QWERTZmods",
```

File: files/mods.json (hunk 5 above)
Comment (QWERTZexe) [reply files/mods.json]: idk, but here it shows "Other"
image

**Comment by QWERTZexe** [comment]
> And re: icon (GH mobile isn't letting me comment on the file)
> 
> This icon is oversensational and out of place compared to other icons, consider replacing it with a more functional one (identifiable at small sizes, static, possibly uses graphic elements)

I added the real icon of the mod. Looks like I accidentaly used the icon of the discord server 😅

**Comment by nacrt** [comment]
i'm pretty sure animated icons wont even work

**Comment by QWERTZexe** [comment]
@KTibow Thanks for optimising the wording!

**Review by MicrocontrollersDev — APPROVED** [review]
seems fine i think

**Comment by QWERTZexe** [comment]
> seems fine i think

And what happens now?

**Comment by MicrocontrollersDev** [comment]
> > seems fine i think
> 
> And what happens now?

someone else should review and merge

**Comment by KTibow** [comment]
This is probably fine for the most part but somebody else must merge

**Comment by QWERTZexe** [comment]
> This is probably fine for the most part but somebody else must merge 

Do I need to ping someone?

**Comment by QWERTZexe** [comment]
@Wyvest I added the keybind :)

**Comment by MicrocontrollersDev** [comment]
will merge tomorrow if no one objects or if no one else merges before me

**Comment by QWERTZexe** [comment]
@Wyvest done

**Pull request merged**